packages/esm-ward-app/src/ward-workspace/admit-patient-form-workspace/admit-patient-form.test.tsx

```diff
@@ -0,0 +1,315 @@
+import React from 'react';
```

Comment:
@denniskigen , requesting you to review this test file, given your recent changes. I hope I am in order of the changes you did recently.
Thanks!